layouts/DefaultLayout.tsx

```diff
@@ -52,13 +45,12 @@ const DefaultLayout: FC<PropsWithChildren> = observer(({ children }) => {
             }
           `}
         </style>
-        {process.env.NEXT_PUBLIC_MIGRATION_IN_PROGRESS !== "true" && <Menu />}
```

Comment:
Is this correct? How does this app look when `process.env.NEXT_PUBLIC_MIGRATION_IN_PROGRESS !== "true"` evaluates to `true`?

Reply:
That was a mistake from a merge. Have since removed it.